spec/models/email_alert_spec.rb

```diff
@@ -89,6 +89,26 @@ def with_lock_unless_done
         "email-alert-api returned unprocessable entity for #{document['content_id']}, #{document['base_path']}, #{document['public_updated_at']}"
       )
     end
+
+    context "for a workflow notification" do
+      before do
+        document["workflow_message"] = "Something changed"
+        email_alert.trigger
+      end
+
+      it "logs receiving a workflow notification for a document" do
+        expect(logger).to have_received(:info).with(
+          "Received workflow message notification for #{document['title']}, with details #{document['details']}"
+        )
+      end
+
+      it "prefers the workflow message to the change note" do
```

Comment:
Are we expecting to either receive a `change_note` or a `workflow_message`? Is it possible to receive both, and should we test for that?

Reply:
The test subject does contain both, though it's not that obvious.
The change note is usually built from the change history which is present here https://github.com/alphagov/email-alert-service/blob/719bd65c579350e9fc536d5d19059060137d8f79/spec/models/email_alert_spec.rb#L20 but we prefer the workflow message if present.